homeassistant/components/broadlink/switch.py

```diff
@@ -159,25 +157,27 @@ def update_data(self):
 
     async def async_added_to_hass(self):
         """Call when the switch is added to hass."""
-        if self._state is None:
+        if self.state is None:
```

Comment:
Is this still needed?

Reply:
I think we can remove the conditional, but not it's body. We need to restore the state of BroadlinkRMSwitch entities.

Reply:
I think we can remove the conditional, but not it's body. We need to be able to restore the state in case assumed_state is True.